syntax = "proto3";

package cache;

service PlayerCacheGRPC {
  rpc updatePlayer(PlayerGrpc) returns (CacheResponse) {}
  rpc getPlayer(Player_id) returns (PlayerGrpc) {}
}

message Player_id { string id = 1; }

message PlayerGrpc {
  string player_id = 1;
  string name = 2;
  int32 hp = 3;
  int32 ac = 4;
  string current_map =5;
}

message CacheResponse { string status = 1; }